app/components/Modal/Modal.tsx

```
@@ -2,43 +1,90 @@
+                  />
+                </svg>
+                <span className="sr-only">Close modal</span>
+              </button>
+            </div>
+            <div className="p-4 md:p-5 space-y-4">
+              <textarea
+                value={text}
+                onChange={(e) => setText(e.currentTarget.value)}
+                placeholder={placeholder}
+                className="placeholder-slate-400 p-1 min-h-300 outline-none focus:ring focus:border-slate-300 dark:bg-gray-600 text-base leading-relaxed text-gray-200 dark:text-gray-200 w-full"
+              ></textarea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)